dt-reports/magic-url-endpoints.php

```diff
@@ -113,6 +113,17 @@ public function email_magic( WP_REST_Request $request ) {
         $errors = [];
         $success = [];
 
+        /**
+         * Bulk Field Filter
+         *
+         * This filter allows for targeting a specific contact connection field to loop through and send emails.
+         * The default field name is 'reporter', but this can be overwritten and pointed towards assigned_to, subassigned_to,
+         * coaching, or another custom contact connection field.
+         *
+         * @param (string)
+         */
+        $target_field = apply_filters( 'dt_bulk_email_connection_field', 'reporter' );
```

Comment:
@ChrisChasm the target_field will depend on the post type. Maybe we can pass that in.
Each magic link might want to specify their own connection field. can we add which magic link is being used as a param too?